apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -6,6 +6,7 @@ var ConversationAccessor = require('./accessors');
 var MessageAccessor = require('./message_accessors');
 var ComposerAccessor = require('../shared/composer_accessors');
 var MenuAccessor = require('../shared/menu_accessors');
+var Tools = require('../shared/tools.js');
```

Comment:
Actually I realised I forgot some isElementFocused in individual tests. Will update.